diffsims/crystallography/reciprocal_lattice_vector.py

```diff
@@ -150,6 +153,53 @@ def __repr__(self):
         phase_name = self.phase.name
         return f"{name} {shape}, {phase_name} ({symmetry})\n" f"{data}"
 
+    @property
+    def basis_rotation(self):
+        """
+        Returns the lattice basis rotation.
+        """
+        return Rotation.from_matrix(self.phase.structure.lattice.baserot)
+
+    @basis_rotation.setter
+    def basis_rotation(self, value):
+        """
+        Sets the lattice basis rotation.
+        """
+        if isinstance(value, Rotation):
+            if value.size != 1:
+                raise ValueError("Rotation must be a single rotation")
+            matrix = value.to_matrix().squeeze()
+        elif isinstance(value, np.ndarray):
+            if value.shape != (3, 3):
+                raise ValueError("Rotation matrix must be 3x3")
+            matrix = value
+        else:
+            raise ValueError("Rotation must be a Rotation or a 3x3 numpy array")
+        self.phase.structure.lattice.setLatPar(baserot=matrix)
```

Comment:
Phases are passed by refrence, so this would affect all other vectors with the same phase.
This is why I copied the phase in `rotate_with_basis`.
```suggestion
        self.phase = self.phase.deepcopy()
        self.phase.structure.lattice.setLatPar(baserot=matrix)
```
The base rotation is not always unitary as well, especially for hexagonal crystals from cif or initialized from a structure. This comes from the enforced x || a, z || c* convention used in orix (https://orix.readthedocs.io/en/stable/tutorials/crystal_reference_frame.html).
What do you intend the `basis_rotation` member to be used for?

Reply:
Fair point. I'll probably just remove this then. I added it for completeness sake but didn't realize that the base rot was used for hexagonal phases.